CODE_OF_CONDUCT.md

```diff
@@ -31,6 +31,50 @@ Examples of unacceptable behavior by participants include:
 - Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
+### Guide to Maintaining these Standards
+
+When interacting in our community, strive to maintain a friendly, empathetic tone. This includes, but is not limited to, interactions on Discord, GitHub, and Twitter.
+
+"Tone" refers to word, punctuation, and conversational choices that determine how your message _feels_ to others.
+
+Compare these two responses:
+
+_Question:_ Hey, I'm beginning with Astro. I tried to use `document` in my Astro page, but it says that the object doesn't exist. How can I manipulate the DOM?
+
+| Answer 1 | Answer 2 |
+| -- | -- |
+| You can't manipulate the DOM from an Astro file. Astro is a static site builder. | Hey, welcome to the community! <br/> Astro pages and components run on the server. Because of that, you won't have access to everything you're used to when writing JavaScript for the web. You're on the right track, though - you can access `document` from a `<script>` tag in your Astro file. <br/> Check out the docs here, and let me know if you need any more help! https://docs.astro.build/en/core-concepts/astro-components/#client-side-scripts |
+
+Answer 2 took more effort, but it made a significant difference. Answer 2
+
+- welcomed the new member into the community
+- used explanation points (conveying willingness to help) and friendly word choices
+- made them feel validated for asking the question ("You're on the right track")
+- provided an explanation that fit the person who asked the question. They specified that they were a beginner, so _Answer 2_ sought to explain things to someone who might not understand the implications of "static site builder".
+- made it clear that the community is willing to help, and asking is not a burden ("let me know if you need any more help!")
+
+Our community prides itself on being _positive_, _helpful_, and _inclusive_. If you have these goals in mind when interacting in the community, you're on the right track!
+
+- To stay positive:
+
+  - Make it clear that you appreciate the other person's message
+  - Use exclamation points and emojis when friendly intention wouldn't be clear otherwise
```

Comment:
```suggestion
```
Hm, I'm not sure recommending this will be helpful. I've noticed some community members abusing emojis to polish a turd, so to speak. This more bubbly approach also isn't _required_ to uphold positivity and inclusivity, especially since it doesn't match everyone's communication style. I'd suggest removing, though you're on the right track 😉

Reply:
Yep, I think I agree here, I’ve also seen “Your website sucks! 🤪” type posts where somehow the emoji is supposed to make this seem like a funny thing.